src/Services/IDatabaseAuditLog.cs

```diff
@@ -0,0 +1,6 @@
+namespace IdentityServerConfig.Services;
+
+public interface IDatabaseAuditLog
```

Comment:
The name of this interface is currently tightly coupled to the actual implementation of writing the audit log to the database. I could imagine having other mechanisms to store this (for example some kind of external service), so perhaps `IAuditLog` would be a more appropriate name?

Reply:
Changed it in 27976c1. I did leave the implementation though, otherwise there would be two AuditLog files